tests/unit/modules/network/eos/test_eos_snmp_server.py

```diff
@@ -151,7 +151,7 @@ def test_eos_snmp_server_merged(self):
             "snmp-server host host01 informs version 3 priv user01 udp-port 23",
             "snmp-server host host02 traps version 2c user01 udp-port 23",
             "snmp-server host host01 informs version 3 priv user01",
-            "snmp-server community comm5 rw list4",
+            "snmp-server community comm6 rw",
```

Comment:
Can we keep a test to make sure both with and without acls work?

Reply:
@NilashishC This particular testcase has a community configuration with acl `snmp-server community comm01 view view3 ro ipv6 list1`